src/Pipe.rei

```diff
@@ -0,0 +1,13 @@
+// Borrowed from ReactMini - the "pipe" (our term) is a mechanism for
+// communicating between instances sharing the same pipe, in the presence of
+// existentials where it's impossible to prove that the two values of the same
+// type actually are of the same type. The pipe allows the value to "coerced"
```

Comment:
Nice description! Just a couple minor typos - `pasing` -> `passing` and `reutnr` -> `return`

Reply:
Thanks. Fixed!